app/views/staff/priorities/new.html.haml

```diff
@@ -1,5 +1,7 @@
 - content_for :page_title_prefix, I18n.t('page_title.staff.schemes.show', name: @scheme.name)
 
+= link_to "Back to #{@scheme.name} scheme view", estate_scheme_path(@scheme.estate, @scheme), class: 'govuk-back-link'
```

Comment:
How about:

```
"Back to scheme #{scheme.name}"
```

This would match the page title created by `I18n.t('page_title.staff.schemes.show', name: @scheme.name`